config.json

```diff
@@ -16,10 +16,15 @@
     "highlightjs_language": "haskell"
   },
   "files": {
-    "solution": [],
-    "test": [],
-    "example": [],
-    "exemplar": []
+    "solution": [
+      "src/%{camel_slug}.u"
```

Comment:
Ah, you've discovered the files template. You likely already know this, but `configlet sync` has a `--filepaths` option that you can use to populate the `files` key in the `.meta/config.json` file of an exercise. For one exercise, the command is:

```shell
configlet sync --update --yes --filepaths --exercise <SLUG>
```

Without the exercise it will update _all_ filepaths.

Reply:
Yeah! 😁  I was wondering how to animate the ``filepaths`` configlet. I'll be sure to use the --exercise flag for now, but in the future I'll probably bring the existing file structure in alignment with this convention!